### PR TITLE
Load correct content in featured-tip container

### PR DESCRIPTION
**Pull request by vendelal**
This PR adds the correct content and adjusts the way other content
is loaded in their respective containers to make sure it always
has a relevant post to the given section. Typographic styling and
padding for links will be included in next PR.

screen shot 2017-10-18 at 3 33 16 pm

### CODE REVIEW

File: source/stylesheets/base/_feature_sections.scss [hunk 1]
```diff
@@ -8,10 +8,11 @@
 }
 
 .jumbo-feature {
-  height: 600px;
+  @include clearfix;
 
   .container {
-    height: inherit;
+    display: flex;
+    justify-content: stretch;
```
Comment (mborsare) [inline source/stylesheets/base/_feature_sections.scss]: This isn't a valid value for `justify-content`. It's for the other one, `align-content`

https://css-tricks.com/almanac/properties/j/justify-content/

File: source/stylesheets/base/_feature_sections.scss [hunk 2]
```diff
@@ -52,6 +52,40 @@
...
+  margin: $base-spacing;
+  text-align: center;
+}
+
+.featured-tip {
+  @include clearfix;
+
+  .container {
+    display: flex;
+    justify-content: stretch;
```
Comment (mborsare) [inline source/stylesheets/base/_feature_sections.scss]: ✂️ cut this

File: source/entries/tips/how-to-add-mixins.html.erb [hunk 3]
```diff
@@ -0,0 +1,18 @@
+---
+
+title: Mixins
+pre_title: How to add
+date: 2017-10-03 20:11 UTC
+layout: recipes
+tags:
```
Comment (mborsare) [inline source/entries/tips/how-to-add-mixins.html.erb]: Can you exclude this if it's empty?

File: source/entries/tips/how-to-add-mixins.html.erb [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+category: tips
+illustration_tag: ../../../images/illustrations/custard.png
+
+---
+
+Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidunt ut laoreet dolore magna aliquam erat volutpat. Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidunt ut laoreet dolore magna aliquam erat volutpat. Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidunt ut laoreet dolore magna aliquam erat volutpat. Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidunt ut laoreet dolore magna aliquam erat volutpat.
```
Comment (mborsare) [inline source/entries/tips/how-to-add-mixins.html.erb:12]: Perhaps set your text editor to wrap lines at 80 characters? It makes multi line paragraphs easy to comment on it Github. I can explain more in person.

File: source/index.html.erb [hunk 5]
```diff
@@ -46,6 +46,26 @@ per_page: 4
   </div>
 </div>
 
+<div class="featured-tip">
+  <div class="container">
+    <% blog.articles.select{ |article| article.data.category == 'tips' }[0...1].each do |article| %>
+      <div class="left">
```
Comment (mborsare) [inline source/index.html.erb]: I'd try and avoid class names like `left` and `right.` They are just too presentational. Maybe go more semantic here or think about abstracting a component you can reuse in the future.

File: source/index.html.erb [hunk 6]
```diff
@@ -46,6 +46,26 @@ per_page: 4
...
+        <h4><%= article.data.pre_title %></h4>
+        <h2><%= article.data.title %></h2>
```
Comment (mborsare) [inline source/index.html.erb]: You might not want to skip `h` levels. Here's some food for thought...

https://www.hobo-web.co.uk/headers/

File: source/index.html.erb [hunk 7]
```diff
@@ -46,6 +46,26 @@ per_page: 4
...
+        <a class="link-light" href="<%= article.url %>">Get More Tips</a>
+      </div>
+      <div class="right">
+        <img src="<%= article.data.illustration_tag %>" class="fadeIn" />
+      </div>
+    <% end %>
+  </div>
+</div>
+
+<div class="drawn-rule">
```
Comment (mborsare) [inline source/index.html.erb]: Perhaps use the `hr` tag? It seems more semantic.

http://htmlreference.io/element/hr/

File: source/stylesheets/base/_feature_sections.scss [hunk 8]
```diff
@@ -52,6 +52,40 @@
   }
 }
 
+.drawn-rule {
+  margin: $base-spacing;
+  text-align: center;
```
Comment (mborsare) [inline source/stylesheets/base/_feature_sections.scss]: Does this do anything?

File: source/stylesheets/base/_feature_sections.scss [hunk 9]
```diff
@@ -52,6 +52,40 @@
...
+      flex-direction: column;
+      justify-content: flex-end;
+    }
+
+    .right {
+      @include grid-column(5 of 12);
+      align-items: center;
+      display: flex;
+
+      img {
```
Comment (mborsare) [inline source/stylesheets/base/_feature_sections.scss]: Your nesting is getting pretty deep here. As a rule of thumb you don't want to go more the 3 levels deep. It starts to get confusing, and overly specific afterwards. I'd really try and keep it as shallow as possible.

File: source/index.html.erb (hunk 6 above)
Comment (vendelal) [reply source/index.html.erb]: The h4 should actually be an h5, but I tried to keep the hierarchy of the h's relative to size. Do you think there's a better way to do it or does this make sense?

screen shot 2017-10-18 at 5 41 37 pm

**Comment by vendelal** [comment]
@mborsare let me know if you think it's good to go!

File: source/entries/tips/how-to-add-mixins.html.erb (hunk 4 above)
Comment (mborsare) [reply source/entries/tips/how-to-add-mixins.html.erb:12]: It doesn't look like this is wrapping?

File: source/stylesheets/base/_feature_sections.scss (hunk 1 above)
Comment (mborsare) [reply source/stylesheets/base/_feature_sections.scss]: Is this still in there?

File: source/index.html.erb [hunk 10]
```diff
@@ -46,6 +46,25 @@ per_page: 4
   </div>
 </div>
 
+<div class="featured-tip">
+  <div class="container">
+    <% blog.articles.select{ |article| article.data.category == 'tips' }[0...1].each do |article| %>
+      <div class="title-and-summary">
+        <h5><%= article.data.pre_title %></h5>
```
Comment (mborsare) [inline source/index.html.erb]: I'd go with a `-` instead of a `_` it's cleaner IMO.

Comment (mborsare) [reply source/index.html.erb]: Or is that a yaml thing? If so nevermind.

File: source/index.html.erb [hunk 11]
```diff
@@ -46,6 +46,25 @@ per_page: 4
...
+        <%= article.summary %>
+        <a class="link-light" href="<%= article.url %>">Get More Tips</a>
+      </div>
+      <div class="illustration">
+        <img src="<%= article.data.illustration_tag %>" class="fadeIn" />
+      </div>
+    <% end %>
+    <svg class="drawn-rule" width="900" height="15" viewBox="0 0 900 15">
+      <path fill="none" stroke="#4A4A4A" stroke-linecap="round" d="M215,2017.47659 C343.855136,2024.52078 324.800214,2024.1308 558.59945,2020.19394 C631.18477,2018.9717 703.592125,2011.84079 776.178598,2010.68906 C889.105063,2008.89724 1002.05953,2011.14001 1115,2011.36548" transform="translate(-215 -2009)"/>
+    </svg>
```
Comment (mborsare) [inline source/index.html.erb]: There's an inline-svg gem that creates a helper for you to use and is a little cleaner. It also makes it easier to add classes to the svg.

Your code would look like `<%= inline_svg("name/of/file.svg" %>`

Check out https://www.snip2code.com/Snippet/692002/Inline-SVG-in-Middleman and try to get it going.

**Review by mborsare — APPROVED** [review]
Small stuff, looking good!

**Pull request merged**